tests/ArrayAccessTest.php

```diff
@@ -1118,7 +1118,7 @@ public function __toString() {
                     function foo(array $a, $key) : void {
                         echo $a[$key];
                     }',
-                'error_message' => 'PossiblyInvalidArrayOffset',
+                'error_message' => 'InvalidArrayOffset',
```

Comment:
This is incorrect, though – since one of the types (`string`) is valid as a key

Reply:
Thanks, it should be better now